src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -802,4 +803,63 @@
     </xsd:complexType>
+
+    <xsd:complexType name="html-sanitizer">
+        <xsd:sequence>
+            <xsd:element name="sanitizer" type="sanitizer" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:attribute name="enabled" type="xsd:boolean" />
+        <xsd:attribute name="default" type="xsd:string" />
+    </xsd:complexType>
+
+    <xsd:complexType name="sanitizer">
+        <xsd:sequence>
+            <xsd:element name="allow-element" type="allow-element" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="block-element" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="drop-element" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="allow-attribute" type="allow-attribute" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="drop-attribute" type="drop-attribute" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="force-attribute" type="force-attribute" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="allowed-link-scheme" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="allowed-link-host" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="allowed-media-scheme" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="allowed-media-host" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="with-attribute-sanitizer" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:sequence>
+            <xsd:element name="without-attribute-sanitizer" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:attribute name="allow-safe-elements" type="xsd:boolean" />
+        <xsd:attribute name="allow-all-static-elements" type="xsd:boolean" />
+        <xsd:attribute name="force-https-urls" type="xsd:boolean" />
+        <xsd:attribute name="allow-relative-links" type="xsd:boolean" />
+        <xsd:attribute name="allow-relative-medias" type="xsd:boolean" />
+    </xsd:complexType>
+
+    <xsd:complexType name="allow-element">
+        <xsd:sequence>
+            <xsd:element name="attribute" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+        <xsd:attribute name="name" type="xsd:string" />
```

Comment:
the attribute holding the map key should be marked as required in the XSD (because it is indeed required in the XML file, *before* merging files together)